ext/date/php_date.c

```diff
@@ -387,6 +387,10 @@ ZEND_BEGIN_ARG_INFO_EX(arginfo_date_period_construct, 0, 0, 3)
 	ZEND_ARG_INFO(0, end)
 ZEND_END_ARG_INFO()
 
+ZEND_BEGIN_ARG_INFO_EX(arginfo_date_set_state, 0, 0, 1)
+	ZEND_ARG_INFO(0, array)
```

Comment:
It would be better to change the name from `array` to `properties` (for parity with the PHP manual on [__set_state](http://php.net/manual/en/language.oop5.magic.php#object.set-state)).

Reply:
I kept `array` because of [DateTime::__set_state()](http://php.net/manual/en/datetime.set-state.php#refsect1-datetime.set-state-parameters), but if would be needed :smile: